src/load/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.3.0
+++++++
+* Add support for autostop criteria. Autostop error rate and time window in seconds can be set using --autostop-error-rate and --autostop-time-window arguments in 'az load test create' and 'az load test update' commands. Autostop can be disabled by using --autostop in 'az load test create' and 'az load test update' commands. Autostop criteria set in YAML config file will now also be honoured.
```

Comment:
Please check if we can add auto-stop criteria in the test runs as well. e.g. I would like to have different auto-stop values in the test run do I need to add the config in the test?

Reply:
@saloniagrawal1997, please review and we can address this in another iteration if needed.